patcher/HitmanPatcher.UI/MainForm.Designer.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace HitmanPatcher
+namespace HitmanPatcher
```

Comment:
[BOM](https://en.wikipedia.org/wiki/Byte_order_mark)/[zwsp](https://en.wikipedia.org/wiki/Zero-width_space)/[nbsp](https://en.wikipedia.org/wiki/Non-breaking_space) added/removed here?

Reply:
Nope, check with Hex editor. I guess it's Githubs way of showing a rename.